internal_docs/review-apps.md

```diff
@@ -0,0 +1,26 @@
+# Review apps for pull requests
+
+This repo uses [Heroku review apps](https://devcenter.heroku.com/articles/github-integration-review-apps)
+to give reviewers a way to easily preview and test changes to the kit.
+
+If you create a pull request from a fork of this repo (typical if the
+contributor is not part of the alphagov organisation) then the review app will
+not be created automatically. However, you can create a review app manually from the
+Heroku dashboard.
+
+Previews of pull requests are automatically published to a URL which has the
+prefix `govuk-prototype-kit` followed by the identifier number of the pull request.
+
+For example, pull request #137 would be deployed to `govuk-prototype-kit-pr-137.herokuapp.com`.
+
+@govuk-design-system-ci should create a GitHub deployment as Heroku builds
```

Comment:
I don't think `govuk-design-system-ci` is really involved in this, other than the fact that Heroku is linked to that account. Suggest something like 'Heroku should create a GitHub deployment when it builds and deploys the review app'?

Reply:
That is technically correct, however it does appear on the face of it that the deployment is done by @govuk-design-system-ci ... I feel like saying 'Heroku creates...' might cause confusion later down the line.

Maybe something like 'As Heroku builds and deploys the review app it will create a GitHub deployment via the @govuk-design-system-ci user'?